### DISCUSSION TITLE
How do you self-host Isso?

### DISCUSSION THREAD

**Discussion by azzamsa** [welpo/tabi#238] [Q&A] [answered]
Hi, Óscar 👋.

How do you self-host Isso? It seems easier to maintain than Webmentions (my currently preferred commenting system), but it requires me to rent a server. I already self-hosted my email (MIAB), but I have to look for a way to set up HTTPS. I have other self-hosted apps, but none of them require HTTPS because they are only personal apps.

Where do you host it? Why did you choose it compared to others?

![image](https://github.com/welpo/tabi/assets/17734314/214c2243-b9e5-4c81-876b-ba355c7bbf8d)

**Comment by welpo** [comment, marked as answer]
Hi!

I host Isso (and GoatCounter) on a super cheap Vultr virtual machine. 2.5$/mo or 3$/mo with automatic backups.

I chose it because of the price. Sadly, the new 2.5$ instances are limited to IPv6. For IPv4 you're looking at 3.5$/mo.

Beyond the price, Vultr has offered impecable service in the 6+ years I've used it. They notify me when there are possible outages and are quick to fix issues and respond to tickets. If you want to give Vultr a try, here's [my referral link](https://www.vultr.com/?ref=9577613-8H) which gets you $100 for 14 days to test them out.

Before that, I used an Oracle's Free Tier instance. It was much more powerful (not that it needs all that power), but Oracle decided to close my account without warning, explanation, or opportunity to recover any data. So I'd advise against that (plus, I found the web interface confusing).

Setting up HTTPS is fairly easy nowadays, thanks to Let's Encrypt. You'll need to set up the DNS records, nginx, and then use certbot. There are many tutorials online for this.

To set up Isso, I followed [this guide](https://blog.phusion.nl/2018/08/16/isso-simple-self-hosted-commenting-system/).
